crates/viewer/re_time_panel/src/lib.rs

```diff
@@ -148,6 +147,12 @@ pub struct TimePanel {
     /// recording with a different application id.
     #[serde(skip)]
     filter_state_app_id: Option<ApplicationId>,
+
+    /// Last clicked item.
+    ///
+    /// We keep track of it to implement the range selection using shift-click.
+    #[serde(skip)]
+    last_clicked_item: Option<Item>,
```

Comment:
same as on the other pr, not happy about the name. But now it's a thing so not feeling strongly about it ;)